ManageIQ/manageiq
# Rest API Git refresh and import

This function allows the Rest API to
- Refresh the contents/branches of the remote repository
- Import a new branch/tag into the Automate domain


Links 
----------------

* https://www.pivotaltracker.com/n/projects/1613499/stories/128530211
* Depends on PR #10298 

Steps for Testing/QA 
-------------------------------

1. Import a domain from a Git Repository using the UI (Automate -> Import/Export)
2. Update the Remote Git Repository with a new branch
3. Use the REST API to refresh the domain and pass in the new branch

This should result in the existing domain getting updated to have the contents of the new branch.
